pkg/detectors/readme/readme.go

```diff
@@ -20,13 +19,13 @@ var (
 	client = common.SaneHttpClient()
```

Comment:
suggestion: Try to use the default client here. `http.DefaultClient`. I think this will resolve the 403. I'm not entirely sure why their API does not allow for our custom client, given it only includes some timeouts and keep lives, but that seems to be the issue.